src/egl_context.c

```diff
@@ -656,8 +732,12 @@ GLFWbool _glfwCreateContextEGL(_GLFWwindow* window,
             _GLFW_OPENGL_LIBRARY,
 #elif defined(_GLFW_WIN32)
 #elif defined(_GLFW_COCOA)
+#else
+#if defined(_GLFW_EGLHEADLESS)
+            "libOpenGL.so.0",
```

Comment:
It's not clear why this is necessary on my system.  It could be glvnd or else my system is misconfigured somehow.